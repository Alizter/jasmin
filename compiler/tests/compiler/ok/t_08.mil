// use indexed variables

fn foo(z.1, z.2 : stack u64) -> stack u64 {
  z.1 += z.2;
  return z.1;
}

/*
START:CMD
ARG="print[input][fun=foo,info]"
END:CMD
*/
