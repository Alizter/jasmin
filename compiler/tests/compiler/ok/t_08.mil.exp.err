Parsing tests/compiler/ok/t_08.mil as MIL

Processed file tests/compiler/ok/t_08.mil
