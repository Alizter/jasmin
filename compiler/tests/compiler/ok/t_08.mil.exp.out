>> input:
fn foo(z.1 : stack u64, z.2 : stack u64) -> stack u64 {
  // START
  z.1 += z.2;
  // END
  return z.1;
}

